contracts/sol6/KyberHintHandler.sol

```diff
@@ -207,6 +215,14 @@ abstract contract KyberHintHandler is IKyberHint, Utils5 {
         bytes32[] memory ethToTokenReserveIds,
         uint256[] memory ethToTokenSplits
     ) public view override returns (bytes memory hint) {
+        if (ethToTokenType == TradeType.BestOfAll) {
+            bytes32[] memory emptyReserveIds;
```

Comment:
can check the input values for Ids and splits are empty
and use them for building hint